src/ex1/script.js

```diff
@@ -0,0 +1,52 @@
+let pendingTodos = [];
+
+function addTodo() {
+  let currentTodo = document.querySelector(".todoInput").value;
```

Comment:
I like how the code is very simple and straightforward, however, I would suggest making it more modular - extracting the different actions into functions will hide the implementation and make the code more clean, readable & maintainable